docs: PCP-6629 + DOC-2740: MAAS SSH + Tags (#10479)
spectrocloud/librarium: docs/docs-content/clusters/data-center/maas/create-manage-maas-lxd-clusters.md

7.  <PartialsComponent category="cluster-templates" name="profile-vs-template" />

8.  <PartialsComponent category="profiles" name="cluster-profile-variables-deployment" />

9.  Fill out the following fields on the **Cluster Config** step.

    | **Field**                  | **Description**                                                                                                                                                                                                                                          |
    | -------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
    | **Domains**                | Register a DNS record in the selected domain for the deployed cluster. The DNS record links to the IP addresses of the control plane nodes.                                                                                                              |

>     | **SSH Keys (Optional)**    | Inject public Secure Shell (SSH) keys into `~/.ssh/authorized_keys` on each cluster node. The list is populated using the keys in **Tenant Settings** or **Project Settings** > **Security** > **SSH Keys**. Select **Add Item** to add additional keys. |

Author: ```suggestion
    | **SSH Keys (Optional)**    | Inject public Secure Shell (SSH) keys into `~/.ssh/authorized_keys` on each control plane and worker node, including LXD VMs. The list is populated using the keys in **Tenant Settings** or **Project Settings** > **Security** > **SSH Keys**. Select **Add Item** to add additional keys. Refer to [SSH Keys](/clusters/cluster-management/ssh/ssh-keys/) for additional information. |
```

>     | **SSH Keys (Optional)**    | Inject public Secure Shell (SSH) keys into `~/.ssh/authorized_keys` on each control plane and worker node, including LXD VMs. The list is populated using the keys in **Tenant Settings** or **Project Settings** > **Security** > **SSH Keys**. Select **Add Item** to add additional keys. Refer to [SSH Keys](/clusters/cluster-management/ssh/ssh-keys/) for additional information. |

Author: ```suggestion
    | **SSH Keys (Optional)**    | Inject public Secure Shell (SSH) keys into `~/.ssh/authorized_keys` on each control plane and worker node, including LXD VMs. The list is populated using the keys in **Tenant Settings** or **Project Settings** > **Security** > **SSH Keys**. Select **Add Item** to [add additional keys](/clusters/cluster-management/ssh/ssh-keys/). |
```

    | **NTP Servers (Optional)** | Specify Network Time Protocol (NTP) servers for the cluster nodes. The servers you provide override the machine image defaults. We recommend specifying at least one NTP server to prevent time drift issues.                                                                                                                              |

> 10. Activate the **Host LXD-Based Control Planes** switch to use this cluster as an LXD-based control plane (hypervisor)

Reviewer: ```suggestion
10. Click the **Host LXD-Based Control Planes** toggle to use this cluster as an LXD-based control plane (hypervisor)
```

Reviewer: Did we discuss this during that UI elements meeting? I can't remember exactly what we decided, but I'm sure we called it a toggle?

Author: Can confirm it should be toggle based on my notes 👍 Thank you!

    | **Minimum CPU**        | Only select machines from the **Resource pool** that have at least the specified amount of CPU.                                                                                                                                                                                                                                                               |
    | **Minimum Memory**     | Only select machines from the **Resource pool** that have at least the specified amount of memory.                                                                                                                                                                                                                                                            |
    | **Availability zones** | The MAAS zones in which to deploy nodes. Palette distributes nodes across the selected zones for high availability.                                                                                                                                                                                                                                           |

>     | **Tags**               | Controls which MAAS machines to use for the node pool based on [MAAS tags](https://canonical.com/maas/docs/about-machine-groups#p-22953-tags). If you specify multiple tags, Palette uses AND logic and only selects machines that have all of the specified tags. <br /> <br /> **WARNING:** Modifying tags on an existing node pool triggers a node repave. |

Author: ```suggestion
    | **Tags**               | Controls which MAAS machines to use for the node pool based on [MAAS tags](https://canonical.com/maas/docs/about-machine-groups#p-22953-tags). If you specify multiple tags, Palette only selects machines that have _all_ of the specified tags. <br /> <br /> **WARNING:** Modifying tags on an existing node pool triggers a node repave. |
```

9.  <PartialsComponent category="profiles" name="cluster-profile-variables-deployment" />

10. Fill out the following fields on the **Cluster Config** step.

    | **Field**                  | **Description**                                                                                                                                                                                                                                          |
    | -------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
    | **Domains**                | Register a DNS record in the selected domain for the deployed cluster. The DNS record links to the IP addresses of the control plane nodes.                                                                                                              |

>     | **SSH Keys (Optional)**    | Inject public Secure Shell (SSH) keys into `~/.ssh/authorized_keys` on each cluster node. The list is populated using the keys in **Tenant Settings** or **Project Settings** > **Security** > **SSH Keys**. Select **Add Item** to add additional keys. |

Author: ```suggestion
    | **SSH Keys (Optional)**    | Inject public Secure Shell (SSH) keys into `~/.ssh/authorized_keys` on each control plane and worker node, including LXD VMs. The list is populated using the keys in **Tenant Settings** or **Project Settings** > **Security** > **SSH Keys**. Select **Add Item** to [add additional keys](/clusters/cluster-management/ssh/ssh-keys/). |
```

    | **Minimum Memory**     | Only select machines from the **Resource pool** that have at least the specified amount of memory.                                                                                                                                                                                                                                                                                                                                                                                 |
    | **Availability zones** | The MAAS zones in which to deploy nodes. Palette distributes nodes across the selected zones for high availability. These must match the same zones used by the hosted LXD-based control plane.                                                                                                                                                                                                                                                                                    |

>     | **Tags**               | Control which MAAS machines to use for the node pool based on [MAAS tags](https://canonical.com/maas/docs/about-machine-groups#p-22953-tags). If you specify multiple tags, Palette uses `AND` logic and only selects machines that have all of the specified tags. If you are using a hosted LXD-based control plane, the resource pool, availability zones, and tags must also match. <br /> <br /> **WARNING:** Modifying tags on an existing node pool triggers a node repave. |

Author: ```suggestion
    | **Tags**               | Control which MAAS machines to use for the node pool based on [MAAS tags](https://canonical.com/maas/docs/about-machine-groups#p-22953-tags). If you specify multiple tags, Palette only selects machines that have _all_ of the specified tags. If you are using a hosted LXD-based control plane, the resource pool, availability zones, and tags must also match. <br /> <br /> **WARNING:** Modifying tags on an existing node pool triggers a node repave. |
```
